src/DacpacTool/PackageInspector.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+
+namespace MSBuild.Sdk.SqlProj.DacpacTool
+{
+    public sealed class PackageInspector
```

Comment:
I decided to make a parallel class to `PackageBuilder` and `PackageDeployer`, but I'm not married to this. The code here could easily be added to `PackageBuilder`, but I thought it would be cleaner to have a class for each possible invocation of `DacpacTool` from MSBuild.

Reply:
Yeah, I was thinking we might not need it all but I'm guessing it makes things a little bit easier to test. I suggested `ScriptInspector` as an alternate name in my review since it doesn't have anything to do with packages.

Reply:
Renamed to `ScriptInspector`.